viewer/packages/rendering/src/glsl/sector/primitives/line.frag

```diff
@@ -0,0 +1,57 @@
+precision highp float;
```

Comment:
Doesn't just `THREE.LineMaterial` work? Why?

Reply:
THREE.LineMaterial is in examples does not come with the library and `THREE.LineBasicMaterial` and `THREE.LineDashedMaterial` line width will not work due to the limitation in the OpenGL core for GL_LINES